source/core/NativeMemory.cs

```diff
@@ -1643,6 +1752,17 @@ public static bool IsModelATrailer(int modelHash)
 			return GetVehicleStructClass(modelInfo) == VehicleStructClassType.Trailer;
 		}
 
+		public static string GetVehicleMakeName(int modelHash)
+		{
+			IntPtr modelInfo = FindCModelInfo(modelHash);
+
+			if (GetModelInfoClass(modelInfo) == ModelInfoClassType.Vehicle)
+			{
+				return PtrToStringUTF8(modelInfo + vehicleMakeNameOffsetInModelInfo);
+			}
+
+			return "CARNOTFOUND";
```

Comment:
Is that a make name recognized by the game? Maybe just return an empty string?

Reply:
I should have explained this. `GET_DISPLAY_NAME_FROM_VEHICLE_MODEL` and `_GET_MAKE_NAME_FROM_VEHICLE_MODEL` (`0xF7AF4F159FF99F97`) returns `CARNOTFOUND` if the model hash cannot be found or is not for vehicle. `Game.GetLocalizedString("CARNOTFOUND")` will return a empty string btw.

Reply:
:+1: